brain/circuit.h

```diff
@@ -61,10 +61,8 @@ class Circuit : public boost::noncopyable
 
     /**
      * @return The set of GIDs for the given target name. If empty it will
```

Comment:
Final conclusion: two functions
- getGIDs(): returns all GIDs of the circuit
- getGIDs( target ): GIDs of given target, throw if target cannot be found (no parsers or invalid target name)